package/Dockerfile.submariner-globalnet

```diff
@@ -4,7 +4,7 @@ ARG TARGETPLATFORM
 WORKDIR /var/submariner
 
 RUN dnf -y install --nodocs --setopt=install_weak_deps=0 \
-           iproute iptables iptables-nft ipset && \
+           iproute iptables iptables-nft ipset grep && \
```

Comment:
Do we need to include grep even on F34? I thought its available by default.

Reply:
> I thought its available by default on F34. Are we including this to be aware that we depend on grep?

It's necessary for proper bump to F36 #1894